docs/User-Guide_Networking.md

```diff
@@ -0,0 +1,106 @@
+# Networking
+
+Armbian uses **Netplan.io** to describe networking configurations. This is the same on minimal IOT, server CLI and desktop images, Debian or Ubuntu based. However, backends are different. 
```

Comment:
Is this actually true for all Armbian images? AFAIK only Ubuntu uses netplan but Debian uses systemd-networkd or something...

Reply:
Yes, its on Debian too. For some time. 

Netplan controls / interact with systemd-networkd or network-manager. You can still use both separately if you like.